.travis.yml

```diff
@@ -1,25 +1,25 @@
 language: rust
-osx_image: xcode8.2
+osx_image: xcode10
 
-rust:
-  - stable
-  - nightly
+git:
+  submodules: true
 
 os:
-  - linux
   - osx
 
-matrix:
-  allow_failures:
-    - rust: nightly
+rust:
+  - stable
```

Comment:
let's pin this to 1.29.2, which is what we've pinned xi-editor to. We'll have to remember to keep these in sync.